server.go

```diff
@@ -240,6 +240,9 @@ func handlePacket(s *Server, p interface{}) error {
 			}},
 		})
 	case *sshFxpOpendirPacket:
+		if _, err := os.Stat(p.Path); err != nil {
```

Comment:
Technically, this would not error if the path requested is not a folder. The spec indicates it must be a directory, so a simple stat isn't complying here. You'd have to make sure the FileInfo.IsDir() is true also.

Reply:
That was a dumb miss on my part. Good catch, thanks.

Reply:
It’s not dumb. It’s a brain fart, and is what code reviews are for! ;)